src/services/authorization-finder.js

```diff
@@ -0,0 +1,31 @@
+const P = require('bluebird');
+const request = require('superagent');
+const ServiceUrlGetter = require('./service-url-getter');
+const ServerResponseHandler = require('./server-response-handler');
+
+function AuthorizationFinder(renderingId, email, password, envSecret) {
+  this.perform = function () {
+    return new P(function (resolve, reject) {
+      const forestUrl = new ServiceUrlGetter().perform();
+
+      request
+        .get(`${forestUrl}/liana/v1/renderings/${renderingId}/authorization`)
+        .set('forest-secret-key', envSecret)
+        .set('email', email)
+        .set('password', password)
```

Comment:
Quite strange to send this in the headers, why don't you do a `POST`?

Reply:
Because on a rest perspective, we want to retrieve (GET) not create (POST). And I don't want to pass the cresentials as query params to avoid leaks in logs. The last viable alternative to me was headers.